src/shared/components/search-result/search-result.component.html

```diff
@@ -31,7 +31,7 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseSta
           </tr>
         </table>
       </th>
-      <th *ngIf="activityEnabled()" style="width: 110px;"></th>
+      <th *ngIf="activityEnabled" style="width: 110px;"></th>
```

Comment:
Maybe move inline styling to css?

Reply:
Good spot. Inherited the code but I should have tidied it up while I was there. 👍

Reply:
Done and pushed.